app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -1,6 +1,11 @@
 module ManageIQ::Providers::EmbeddedAnsible::CrudCommon
   extend ActiveSupport::Concern
 
+  include do
```

Comment:
Think this is `included`

```suggestion
  included do
```